installation problem

I have tried to install the released version of the package (0.1) with GAP 4.10.2,
which should work, according to the dependencies in `PackageInfo.g`.
I got the following error messages.

```
  CC       src/meataxe64_la-meataxe64.lo
src/meataxe64.c: In function ‘FuncMTX64_Make8BitConversion’:
src/meataxe64.c:689:59: error: macro "FFE_FELT_FIELDINFO_8BIT" passed 2 arguments, but takes just 1
       FFV ent2 = VAL_FFE(FFE_FELT_FIELDINFO_8BIT(info,ent));
                                                           ^
In file included from /users/sam/gap/gap-4.10.2/src/compiled.h:30:0,
                 from src/meataxe64.h:5,
                 from src/meataxe64.c:12:
src/meataxe64.c:689:26: error: ‘FFE_FELT_FIELDINFO_8BIT’ undeclared (first use in this function)
       FFV ent2 = VAL_FFE(FFE_FELT_FIELDINFO_8BIT(info,ent));
                          ^
```

The same happens when I try to install the master branch of the package with GAP 4.10.2.

(The installation of the master branch version of the package with the master branch of GAP works.
Is the package meataxe64 really expected to work with GAP 4.10?)

P.S.:
The installation instructions in `README.md` should say `with-gaproot` not `with-gap-root`.

--- a/PackageInfo.g
+++ b/PackageInfo.g
@@ -87,7 +87,7 @@ PackageDoc := rec(
 ),
 
 Dependencies := rec(
-  GAP := ">= 4.10",
+  GAP := ">= 4.11",
                    NeededOtherPackages := [ [ "GAPDoc", ">= 1.5" ], 
                            [ "AutoDoc", ">= 0.0.0"],
                           [ "datastructures", ">= 0.0.0"] ],

